tools/scripts/set-up-e2e-fixtures.js

```diff
@@ -0,0 +1,53 @@
+const shell = require('shelljs');
+const tmp = require('tmp').dirSync().name;
+const path = require('path');
+const fs = require('fs');
+
+if (fs.existsSync(path.join('tmp', 'proj', 'angular.json'))) {
+  console.log('Reusing existing E2E test fixtures...');
+} else {
+  console.log('Setting up E2E test fixtures...');
+  shell.rm('-rf', 'tmp');
+  shell.mkdir('tmp');
+
+  shell.mkdir(path.join(tmp, 'proj'));
+  shell.mkdir(path.join(tmp, 'ng'));
+
+  shell.exec('yarn add @angular/cli@7.3.6', { cwd: path.join(tmp, 'ng') });
```

Comment:
Maybe we should use a later version (I realize this is just our old setup fixtures script)